### ISSUE TITLE
Improve documentation defaults when swapping from yii2-swiftmailer

### ISSUE THREAD

**Issue by marcovtwout** [yiisoft/yii2-symfonymailer#33]
When swapping from deprecated https://github.com/yiisoft/yii2-swiftmailer to this extension, two things needed to be changed to keep everything working as before:

1. Swiftmailer default transport was the SendmailTransport, while this extension will default to a NullTransport (sends no mail). You can use the swiftmailer default by defining in application config:
```
'mailer' => [
    'class' => yii\symfonymailer\Mailer::class,
    'transport' => [
        'dsn' => 'sendmail://default',
    ],
],
```

2. Codeceptions TestMailer specifies swiftmailer\Message as a default class in https://github.com/Codeception/module-yii2/blob/master/src/Codeception/Lib/Connector/Yii2/TestMailer.php#L8. This can also be changed by defining in test config:

```
'mailer' => [
    'class' => yii\symfonymailer\Mailer::class,
    'messageClass' => \yii\symfonymailer\Message::class,
],
```

Not sure if any of these should be a code change, but I thought it was useful to report and perhaps have it documented in the README.

**Comment by samdark** [comment]
Yes, would be super-useful in a readme. Would you please make a pull request?

**Comment by marcovtwout** [comment]
I'm really busy at the moment, but will revisit this ticket later.

**Comment by mcserep** [comment]
After this PR was merged, commit 716243e9e8250c71e30280e287dbcd7ff9f2507f removed the recommended migration for `config/test.php` from the README.

I think it should readded, because in case `'messageClass' => \yii\symfonymailer\Message::class`  is not specified there, CodeCeption fails, as it tries to use SwiftMailer by default.
